style.css

```diff
@@ -338,3 +356,111 @@ textarea {
 .get-in-touch:active {
   background-color: #f3f3f3;
 }
+
+.d-none {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .main-container {
+    background-image: url('./images/headerbg.svg');
+    align-items: center;
+    justify-content: center;
+    height: 93vh;
+  }
+
+  .header-container {
```

Comment:
- [ ] Give the navigation/header a little more padding on the left and right so that it matches the template's design.
Your design
![Screenshot from 2021-08-19 12-02-48](https://user-images.githubusercontent.com/65501535/130040955-edfbe4bc-8e04-4bd1-94a1-696ddb0449d1.png)

Original design
![Screenshot from 2021-08-19 12-03-37](https://user-images.githubusercontent.com/65501535/130041105-bc2cacf0-2742-41c1-92a6-e1da8b0b739a.png)